src/app/components/DeleteButton/DeleteButton.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from 'styled-components';
+
+type DeleteButtonProps = {
+  handleDelete: () => void;
+  content: string;
```

Comment:
Maybe you want to use `children` here, that way you could add an SVG to display on the button in the future

Reply:
Very good idea, thank you!